### DISCUSSION TITLE
Issue with Bitmap object

### DISCUSSION THREAD

**Discussion by SignalMQ** [AvaloniaUI/Avalonia#13866] [Q&A]
## Describe the bug
I tried to create Avalonia.Media.Imaging.Bitmap and got an exception when initializing the object.

## To Reproduce

1. Create a console project with `.NET 8.0` and install the `Avalonia 11.0.6` nuget package.
2. Use namespace with `using Avalonia.Media.Imaging;`
3. Create a new `Bitmap` object with parameter initialization.
4. Build and run project

## Expected behavior
Bitmap initialized successfully without any exceptions.

## Screenshots
![image](https://github.com/AvaloniaUI/Avalonia/assets/77734001/b66c89c0-ecca-451a-a295-45cfede879c1)

## Environment 

- OS: Kubuntu 23.10 with Linux 6.6.4 (builded manually)
- Avalonia-Version: 11.0.6

## Additional context
![image](https://github.com/AvaloniaUI/Avalonia/assets/77734001/ad16759b-3038-4b6e-8937-c5d1691afb94)

```
Unable to locate 'Avalonia.Platform.IPlatformRenderInterface'.
   at Avalonia.LocatorExtensions.GetRequiredService[T](IAvaloniaDependencyResolver resolver)
   at Avalonia.Media.Imaging.Bitmap.GetFactory()
   at Avalonia.Media.Imaging.Bitmap..ctor(String fileName)
   at Program.<Main>$(String[] args) in /home/sobirjon/Documents/Console/Program.cs:line 3
```

**Comment by Gillibald** [comment]
Avalonia objects require an initialized application

**Comment by timunie** [comment]
@SignalMQ I think this is by desing. You need to wait for Avalonia to finish loading before you create a Bitmap. I'll make a Q&A from this for now.
